include/grpc++/impl/codegen/status.h

```diff
@@ -46,6 +46,16 @@ class Status {
         error_message_(error_message),
         binary_error_details_(error_details) {}
 
+  /// Construct an instance with \a code,  \a error_message and
+  /// \a error_details. It is an error to construct an OK status with non-empty
+  /// \a error_message and/or \a error_details.
+  Status(StatusCode code, const grpc::string& error_message,
```

Comment:
Maybe this should be private or removed. A user may set a Status with this at server side and expect to see the same error_string at the client side...

Reply:
Negated by comment below